### PR TITLE
Update custom-types.json

### PR DESCRIPTION
**Pull request by nightohl**
- '.hwp'(hangul word processor) is the most popular document type in Korea.
- hwp and related extensions are added in custom-types.json

### CODE REVIEW

File: src/custom-types.json [hunk 1]
```diff
@@ -890,5 +890,27 @@
     "sources": [
       "https://developer.mozilla.org/en-US/docs/Web/WebGL/Adding_2D_content_to_a_WebGL_context"
     ]
+  },
+  "application/vnd.hancom.hwp": {
+    "extensions": ["hwp"],
+    "notes": ".hwp (Hangul Word-Processor) is the Most popular document type in Korea.",
+    "sources": [
+      "https://cdn.hancom.com/link/docs/%ED%95%9C%EA%B8%80%EB%AC%B8%EC%84%9C%ED%8C%8C%EC%9D%BC%ED%98%95%EC%8B%9D_5.0_revision1.3.pdf"
```
Comment (dougwilson) [inline src/custom-types.json:898]: I'm not sure if I'm missing it, but I don't see any reference to the MIME type `application/vnd.hancom.hwp` in this document. Can you just let me know which page number it is on?

File: src/custom-types.json [hunk 2]
```diff
@@ -890,5 +890,27 @@
...
+    ]
+  },
+  "application/vnd.hancom.hml": {
+    "extensions": ["hml"],
+    "notes": "HWPML (hwp + xml) document structure.",
+    "sources": [
+      "https://cdn.hancom.com/link/docs/%ED%95%9C%EA%B8%80%EB%AC%B8%EC%84%9C%ED%8C%8C%EC%9D%BC%ED%98%95%EC%8B%9D3.0_HWPML_revision1.2.pdf"
```
Comment (dougwilson) [inline src/custom-types.json:905]: I'm not sure if I'm missing it, but I don't see any reference to the MIME type `application/vnd.hancom.hml` in this document. Can you just let me know which page number it is on?

File: src/custom-types.json [hunk 3]
```diff
@@ -890,5 +890,27 @@
...
+    ]
+  },
+  "application/vnd.hancom.owpml": {
+    "extensions": ["owpml"],
+    "notes":".owpml (Open Word-Processor Markup Language) is national standard version of HWPML in Korea, and it is supported from the HWP 2018 version.",
+    "sources": [
+      "https://standard.go.kr/KSCI/standardIntro/getStandardSearchView.do?ksNo=KSX6101",
```
Comment (dougwilson) [inline src/custom-types.json:912]: I'm not sure if I'm missing it, but I don't see the MIME type `application/vnd.hancom.owpml` anywhere on this page.

File: src/custom-types.json [hunk 4]
```diff
@@ -890,5 +890,27 @@
...
+      "https://e-ks.kr/streamdocs/view/sd;streamdocsId=72059197557727331"
```
Comment (dougwilson) [inline src/custom-types.json:913]: I'm not sure if I'm missing it, but I don't see any reference to the MIME type `application/vnd.hancom.owpml` in this document. Can you just let me know which page number it is on?

**Review by dougwilson — CHANGES_REQUESTED** [review]

**Comment by nightohl** [comment]
Sorry, the attached source is a document format link.
Those mimtypes not specified on the website, but The program released for Linux specifies mimetypes.
I will check the contents and PR again.

**Pull request closed without merging**